### ISSUE TITLE
Support for [enum] in forms rendered from values.json.schema

### ISSUE THREAD

**Issue by batiati** [kubeapps/kubeapps#1721]
Render values defined in "enum" constraint as a selection box on forms

Example of values.json.schema
```
    "databaseType": {
      "type": "string",
      "form": true,
      "enum": ["mariadb", "postgresql"],
      "title": "Database Type",
      "description": "Allowed values: \"mariadb\" and \"postgresql\""
    }
```

Current behavior: it is rendered as free text

**Comment by andresmgot** [comment]
Good point, thanks for the feature request @batiati. This is kind of related to https://github.com/kubeapps/kubeapps/issues/1715 but different. Will take into account :+1:

**Comment by batiati** [comment]
Hi folks,
I implemented the selection box support for enum values. As my charts use this feature extensively it was very important to improve the usability.

I'd like to contribute with the code of this feature, but I don't know exactly how ... Is it just me make a pull request, or do I need to do something else? Perhaps there are some docs to update too, right? 
 
![Screenshot from 2020-06-07 13-54-40](https://user-images.githubusercontent.com/5728002/83974842-9715ca00-a8c6-11ea-8c1f-83afc17c24b7.png)

**Comment by andresmgot** [comment]
Hi @batiati,

Yes a PR would be great! There is also some documentation around this here: https://github.com/kubeapps/kubeapps/blob/master/docs/developer/basic-form-support.md so in case there is a new render type we would need to add it there as well.

**Comment by absoludity** [comment]
Fixed in #1721
